api/lib/infrastructure/repositories/certification-center-membership-repository.js

```diff
@@ -49,6 +54,34 @@ function _toDomain(certificationCenterMembershipDTO) {
   });
 }
 
+const countAdminMembersForCertificationCenter = async function (certificationCenterId) {
```

Comment:
Je suggère le nommage suivant car « AdminMembers » semble vraiment trop surchargé de sens : 

```suggestion
const countAdminsForCertificationCenter = async function (certificationCenterId) {
```

Si cette modification de nommage est acceptée, alors il faut faire des remplacements `AdminMembers`→`Admins` et `adminMembers`→`admins` dans tous les commits de cette PR.